Sources/MapboxNavigation/CPTrip.swift

```diff
@@ -3,8 +3,8 @@ import CarPlay
 
 @available(iOS 12.0, *)
 extension CPTrip {
-    convenience init(routes: [Route], routeOptions: RouteOptions, waypoints: [Waypoint]) {
-        let routeChoices = routes.enumerated().map { (routeIndex, route) -> CPRouteChoice in
+    convenience init(routeResponse: RouteResponse, routeOptions: RouteOptions, waypoints: [Waypoint]) {
```

Comment:
RouteResponse has an `options` property that should reflect the options in the request. Do we need to keep a separate `routeOptions` argument around, or can we assume it’s always the same options?

Reply:
It is expected to be same options.
Previously, we limited this `CPTrip.userInfo` to contain exactly `RouteOptions` object. It is not documented anywhere or, exposed publicly (or at least I could not find any public promise made around it). Technically, we could use `routeResponse.options` and cover both `RouteOptions` and `MatchOptions` cases. 
It seems rather safe change to me, so I'll update it if there's no objections.

Reply:
Looks like same applies to `waypoints` argument. It is extracted from the route response